### ISSUE TITLE
Update mongoose

### ISSUE THREAD

**Issue by fenos** [typegoose/typegoose#579]
Hello, when i run my service i get:

```
  console.warn
    Using Unsupported mongoose version, highest supported is 5.10.18 (Current version: 5.12.11)
```

Can you please update `typegoose` to use the latest mongoose. That warning is super annoying... 🙏

**Comment by smolinari** [comment]
@fenos - You'd need to understand that the types for Mongoose changed after 5.10.18 (version 5.10.19 and above) and basically broke a lot of stuff. So, Typegoose (v7) had to fix its version to 5.10.18. What happened is, the Mongoose team decided to write their own types and drop the need for `@types/mongoose`.  

That being said, there is now version 8 of Typegoose in beta, which uses the newest versions of Mongoose. So, if you want to use newer versions of Mongoose than 5.10.18, you need to upgrade to version 8 (and that should answer your question). 

https://github.com/typegoose/typegoose/releases/tag/v8.0.0-beta.21

Scott

**Comment by hasezoey** [comment]
Duplicate of #432
